doc/TRAINING.rst

```diff
@@ -46,22 +46,15 @@ Install the required dependencies using ``pip3``\ :
 .. code-block:: bash
 
    cd DeepSpeech
-   pip3 install -e .
+   pip3 install --upgrade pip wheel setuptools
```

Comment:
Do we need latest version ?

Reply:
I don't think so, but depending on the OS/distro/Python version fresh virtualenvs come with older versions so for users of the training code I added this instruction to be safe.

Reply:
For example some versions are incompatible with TF 1.15 wheels.

Reply:
Sure, but forcing always latest version can bring its own issues. I'd feel better if we could stick to well known versions, like we do at several other places.